### DISCUSSION TITLE
CVEs exist in the software that airflow relies on

### DISCUSSION THREAD

**Discussion by diandian-monster** [apache/airflow#17246] [Q&A] [answered]
1、graphviz 0.14.2 CVE-2020-18032 
2、thrift 0.13.0 CVE-2020-13949

**Comment by potiuk** [comment, marked as answer]
Does it have some exploitable paths? Is it dangerous? What are the ways to use those? There are many tools that are reporting CVEs in dependencies, but it does not automatically mean that the software using them is also vulnerable.


Also, If you think those are exploitable issues, there is a process in ASF https://www.apache.org/security/  to report the vulnerabilities in responsible way. Please do so in case you have a reproduction/exploitability case and it will be handled in responsible way.

It's a VERY BAD idea to make public statements about CVEs (and especially bad if you don't even know if they are valid or not as this creates a lot of noise.

Please follow https://www.apache.org/security/ if you want to report security issues.
